documentation/configuration.md

```diff
@@ -60,7 +60,7 @@ In order to have perseo running, there are several basic pieces of information t
 * `config.smpp.password`: Password for the user of the SMPP server
 * `config.smpp.from`: Number from SMS are sending by SMPP server
 * `config.smpp.enabled`: SMPP is default method for SMS instead of use SMS gateway.
-* `config.orion.URL`: URL for updating contexts at Orion (Context Broker).
+* `config.orion.URL`: Context Broker URL, e.g. https://orion.example.com
```

Comment:
This PR, needs a change in the configuration that we need to validate.

The URL for the updateContext can not include the path, since it is not the same for NGSIv2. So we propose to change this variable by the URL of the context Broker and add the path depending on whether the update is v1 or v2.

Reply:
After internal discussion we think is a good idea to redefine the semantic of `config.orion.URL` (and associated PERSEO_ORION_URL env var) to be the base URL of Orion (so it gets complented by `/v1/updateContext` in the case of NGSIv1 update action and `/v2/entities?options=upsert` in the case of NGSIv2 update action).

However, I'd suggest to use "base URL" intead of "URL" (in can be confussing) and use an example closer to a real Orion (i.e. using port 1026). In sum:

```
`config.orion.URL`: Context Broker base URL, e.g. https://orion.example.com:1026
```

Reply:
Fixed in d30f1ed